manager/src/test/java/com/redhat/service/bridge/manager/dao/ProcessorDAOTest.java

```diff
@@ -164,7 +165,7 @@ public void findByBridgeIdAndCustomerId() {
     @Test
     public void findByBridgeIdAndCustomerId_noProcessors() {
         Bridge b = createBridge();
-        ListResult<Processor> listResult = processorDAO.findByBridgeIdAndCustomerId(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, 0, 100);
+        ListResult<Processor> listResult = processorDAO.findByBridgeIdAndCustomerId(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, new QueryInfo(0, 100));
         assertThat(listResult.getPage()).isZero();
```

Comment:
I would not use 0 and 100 since they have little evident semantics. Possibly you could use a constant like PAGE_0 and PAGE_SIZE_LARGE instead?